5.3.the_wisper_woman.py

```diff
@@ -0,0 +1,42 @@
+import os.path
+import re
+
+
+def search_file_for_strings(file_path):
+    """
+    The function reads in a binary file at the provided file path
+    and searches for secret messages within it.
+    The secret messages are defined as strings of lowercase English letters that are at
+    least 5 characters long and end with an exclamation point.
+    To search for the secret messages, the function reads in the file in blocks of 1024 bytes,
+    converts each block to a lowercase string containing only letters, and uses regular expressions
+    to search for the secret messages within that string. The function yields each secret message found.
+
+    :param file_path: 'logo.jpg' image path.
+    :return: one secret message at a time.
+    """
+    with open(file_path, 'rb') as f:
+        while True:
```

Comment:
I would prefer to see the "if not block" implemented as the while-boolean-statement, but also ok